org.freeplane.App.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.freeplane.App
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+  - --allow=multiarch
```

Comment:
What is it needed for?

Reply:
You mean the multiarch thing? I though it was convenient for java multiplatform apps but seems I was wrong.